.github/workflows/ci_cd.yml

```
@@ -637,7 +637,9 @@ jobs:
 
           It looks like some of the Playwright tests failed. You can download the trace output and image diffs for the failed tests under the "Artifacts" section in the above page.
 
-          You can read the [docs on how to use this artifact](https://docs.openverse.org/guides/frontend/playwright_tests.html#debugging).
+          You can read the [docs on how to use this artifact](https://docs.openverse.org/frontend/guides/playwright_tests.html#debugging).
```

Should this include `reference` and remove `guides` from the URL? It's not working for me otherwise. See the preview:
- working link: https://docs.openverse.org/_preview/1975/frontend/reference/playwright_tests.html#debugging
- not working: https://docs.openverse.org/frontend/guides/playwright_tests.html#debugging

```suggestion
          You can read the [docs on how to use this artifact](https://docs.openverse.org/frontend/reference/playwright_tests.html#debugging).
```